README.md

```diff
@@ -1,3 +1,11 @@
+[![Project Status: Unsupported – The project has reached a stable, usable state but the author(s) have ceased all work on it. A new maintainer may be desired.](https://www.repostatus.org/badges/latest/unsupported.svg)](https://www.repostatus.org/#unsupported)
```

Comment:
I don't think we desire a new maintainer, otherwise good.

Reply:
true. removed it (it's the standard text of that badge, but in our case we are not even looking for one anymore).